companies/serializers.py

```
@@ -4,6 +4,11 @@
 from .models import Company
 
 class CompanySerializer(serializers.ModelSerializer):
+    rank = serializers.SerializerMethodField()
     class Meta:
         model = Company
-        fields = ['ticker_symbol', 'company_name', 'description', 'rank_in_industry']
\ No newline at end of file
+        fields = ['ticker_symbol', 'company_name', 'description', 'rank']
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)